discord/utils.py

```diff
@@ -1067,6 +1069,14 @@ def set_target(
                 except AttributeError:
                     pass
 
+
+def _generate_session_id() -> str:
```

Comment:
Wait, does this just generate a random string of Characters?

Reply:
Indeed. It isn't really validated.